pkg/kubelet/eviction/api/types.go

```diff
@@ -36,6 +36,8 @@ const (
 	SignalImageFsAvailable Signal = "imagefs.available"
 	// SignalImageFsInodesFree is amount of inodes available on filesystem that container runtime uses for storing images and container writeable layers.
 	SignalImageFsInodesFree Signal = "imagefs.inodesFree"
+	// SignalAllocatableMemoryAvailable is amount of memory available for pod allocation (i.e. allocatable - workingSet (of pods), in bytes.
+	SignalAllocatableMemoryAvailable Signal = "allocatableMemory.available"
```

Comment:
Just to clarify, in-case I wasnt clear:  This is now in a separate package from the rest of the eviction code, so I have 2 options:
1. I can move this signal (or all of them) to eviction/types.go, and make it private.  However, then it is separated from the rest of the signals.
2. I can leave it public.

I am happy to do either, just let me know.